alma/common/hpc-tuning.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
 # Disable some unneeded services by default (administrators can re-enable if desired)
+systemctl disable wpa_supplicant
```

Comment:
Just curious why the additional services need to be disabled?

Reply:
> Just curious why the additional services need to be disabled?

this was one of my questions for you about the differences in the existing files. I took this from the existing alma 8.x hpc-tuning.sh